app/src/main/java/de/westnordost/streetcomplete/quests/surface/RoadSurface.kt

```diff
@@ -2,28 +2,30 @@ package de.westnordost.streetcomplete.quests.surface
 
 import de.westnordost.streetcomplete.R
 import de.westnordost.streetcomplete.data.meta.ANYTHING_UNPAVED
-import de.westnordost.streetcomplete.data.meta.updateWithCheckDate
-import de.westnordost.streetcomplete.data.osm.osmquest.SimpleOverpassQuestType
 import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder
 import de.westnordost.streetcomplete.data.osm.mapdata.OverpassMapDataAndGeometryApi
+import de.westnordost.streetcomplete.data.osm.osmquest.SimpleOverpassQuestType
 import de.westnordost.streetcomplete.settings.ResurveyIntervalsStore
 
 
-class AddRoadSurface(o: OverpassMapDataAndGeometryApi, r: ResurveyIntervalsStore)
-    : SimpleOverpassQuestType<String>(o) {
-
+class RoadSurface(o: OverpassMapDataAndGeometryApi, r: ResurveyIntervalsStore) : SimpleOverpassQuestType<DetailSurfaceAnswer>(o) {
     override val tagFilters = """
         ways with highway ~ ${ROADS_WITH_SURFACES.joinToString("|")}
-         and (
-           !surface
-           or surface ~ ${ANYTHING_UNPAVED.joinToString("|")} and surface older today -${r * 4} years
-           or surface older today -${r * 12} years
-         )
-         and (access !~ private|no or (foot and foot !~ private|no))
+        and
+        (
+            !surface
+            or surface ~ ${ANYTHING_UNPAVED.joinToString("|")} and surface older today -${r * 4} years
+            or surface older today -${r * 12} years
+        or
+            (
+            surface ~ paved|unpaved
+            and !surface:note
+            and segregated != yes
+            )
+        )
+        and (access !~ private|no or (foot and foot !~ private|no))
     """
-    /* ~paved ways are less likely to change the surface type */
-
-    override val commitMessage = "Add road surfaces"
+    override val commitMessage = "Add road surface info"
```

Comment:
Won't this upset the stats system?

Reply:
AFAIK no, stats read changeset tags - not changeset comments.

And app/src/main/java/de/westnordost/streetcomplete/data/user/achievements/AchievementsModule.kt  is modified to handle that detail quest is merged into surface quest.

Reply:
correct